SECURITY.md

```diff
@@ -4,8 +4,8 @@
 
 | Version | Supported          |
 | ------- | ------------------ |
-| 6.x     | :white_check_mark: |
-| <6.x    | :x:                |
+| 7.x     | :white_check_mark: |
+| <7.x    | :x:                |
```

Comment:
Can we say `>= 6.0` as supported, and `< 6.0` as unsupported? I think it makes sense to support the last minor version.

Reply:
In that case, another PR should be pushed that changes the current version to 5.0.

Reply:
> In that case, another PR should be pushed that changes the current version to 5.0.

There's not really a reason to do that since we're soon to release 7.0

Reply:
7.0 still has a ways to go. If you view the milestone, 16 issues or PRs still exist, and I'd like to type all the modules. This depends on PR #1381 because if it isn't pushed, then I will have to move around a lot of code in the Subreddit file.

Reply:
I don't think we should wait for everything to be finished. We have a ton of outstanding changes. I'd rather release 7 and move stuff to 8 so we can get the last months of changes out there. Some PRAW major releases have only existed for a few weeks, and that's fine.